source/com.microsoft.tfs.core/src/com/microsoft/tfs/core/telemetry/TfsTelemetryHelper.java

```diff
@@ -29,8 +30,13 @@
     private static final Log log = LogFactory.getLog(TfsTelemetryHelper.class);
 
     private static TelemetryClient aiClient;
+    protected static boolean sendingTelemetryDisabled = false;
 
-    static {
+    // A property to define whether collecting telemetry should be skipped.
+    private static final String DISABLE_TELEMETRY_PROPERTY_NAME = "com.microsoft.tfs.core.telemetry.disabletelemetry"; //$NON-NLS-1$
+    private static final String DISABLE_TELEMETRY_ENVIRONMENT_VAR = "disabletelemetry"; //$NON-NLS-1$
```

Comment:
May we move this constant to the EnvironmentVariables class?
To be consistent, may we define the constant as DISABLE_TELEMETRY = "TF_DISABLE_TELEMETRY"